Gemfile

```diff
@@ -31,7 +33,7 @@ gem 'jquery-ui-rails', '~> 4.2.0'
 group :development, :test do
   gem 'rspec-rails'
   gem 'shoulda'
-  gem 'capybara'
+  gem 'capybara', '3.36.0' # Last version to support Ruby 2.6
```

Comment:
We should actually upgrade the container to use Ruby 2.7, but it can wait. Ruby 2.7 has been used in production.